application/src/main/java/dev/ikm/komet/app/util/CssFile.java

```diff
@@ -0,0 +1,119 @@
+package dev.ikm.komet.app.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Enum representing supported CSS files along with their associated metadata.
+ * <p>
+ * Each enum constant encapsulates the CSS file name, module name, resource path within the module
+ * and the base directory for resources.
+ * </p>
+ * <p>
+ * To add a new CSS file, declare a new enum constant with the appropriate metadata.
+ * Ensure that the resource path uses forward slashes (/) and matches the actual file location.
+ * </p>
+ *
+ * @see CssUtils
+ */
+public enum CssFile {
+
+    KOMET_CSS(
+            "komet.css",
+            "dev.ikm.komet.framework",
+            "dev/ikm/komet/framework/graphics/komet.css",
+            Paths.get("framework", "src", "main", "resources")
+    ),
+
+    KVIEW_CSS(
+            "kview.css",
+            "dev.ikm.komet.kview",
+            "dev/ikm/komet/kview/mvvm/view/kview.css",
```

Comment:
More of a question: since the file is on line 29, should line 31 contain the file name?

Reply:
Correct, we can re-use the file name from line 29 to line 31. Making the required change.